<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionPlayBGM</class>
 <widget class="QWidget" name="ActionPlayBGM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblIsStop">
       <property name="text">
        <string>Stop:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="isStop"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblMusic">
       <property name="text">
        <string>Music:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblFadeInDelay">
       <property name="text">
        <string>Fade in Delay:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="fadeInDelay">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblFadeOutDelay">
       <property name="text">
        <string>Fade Out Delay:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="fadeOutDelay">
       <property name="minimum">
        <number>-2147483647</number>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ObjectSelector" name="music" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectSelector</class>
   <extends>QWidget</extends>
   <header>ObjectSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
